gsoc/2017/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Google Summer of Code 2017"
+author: "Benedikt Hegner"
+layout: default
+year: 2017
+redirect_from: /gsoc/gsoc2017.html
+---
+
+# ![CERN](/images/CERN-logo.jpg){:height="100px"} ![HSF](/images/hsf_logo_angled.png){:height="100px"} Google Summer of Code 2017
+
+## Introduction
+
+[Google Summer of Code](https://developers.google.com/open-source/gsoc/) is a program that allows students to contribute to development of open-source projects, mentored by participating organizations.
```

Comment:
The 3 first § should be removed as they duplicate the new GSOC introduction in `gsoc/index.md`.